modules/adagioAnalyticsAdapter.js

```diff
@@ -257,11 +265,33 @@ function handlerBidResponse(event) {
     return;
   }
 
+  if (!event.pba) {
+    return;
+  }
+
   cache.updateAuction(event.auctionId, event.adUnitCode, {
-    adg_sid: event.seatId || null
+    ...event.pba
```

Comment:
I'm wondering if we can do something like: 

```js
var x = {...Object.keys(pba).reduce((acc, key) => {
   const normalizedKey = (key.startsWith('adg_')) ? key : `adg_${key}`
   acc[normalizedKey] = pba[key];
    return acc;
  }, {})};
```

This way we can save bytes on each `response.pba` properties sent from the engine.

Reply:
![](https://media.giphy.com/media/dsod8IVMwjLEvmFeF1/giphy.gif)

Reply:
Plus we normalise the prefix `adg` on each dynamic qp linked for the bid from the engine

Reply:
😆